src/jvmTest/kotlin/com/charleskorn/kaml/JvmYamlReadingTest.kt

```diff
@@ -42,5 +42,23 @@ class JvmYamlReadingTest : DescribeSpec({
                 result shouldBe 123
             }
         }
+
+        describe("parsing into a YamlNode from a string") {
+            val input = "123"
+            val result = Yaml.default.parseToYamlNode(input)
+
+            it("successfully deserializes values from a string") {
+                result.yamlScalar.toInt() shouldBe 123
```

Comment:
These two tests are testing a lot more than just parsing to a `YamlNode` - they're also testing the `yamlScalar` extension property and integer conversion. Is there a way to narrow the scope of the test to just focus on the parsing? 

(You've already added tests to cover the other aspects, which is great by the way 🎉)

Reply:
Done.